readme.md

```diff
@@ -152,14 +157,12 @@ This repository contains [provisioning service client SDK](./provisioning/servic
 
 > Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
 
-# Long Term Support
+# Long-Term Support (LTS)
 
 The project offers a Long Term Support (LTS) version to allow users that do not need the latest features to be shielded from unwanted changes.
 
-As of August 2020, the .NET SDK is shifting to a revised Long-Term Support strategy. The primary motivations for this change are to extend the support period and decrease the churn on LTS releases, while still maintaining a strategy that offers customers choice between new features and stability. 
-
-We now will be releasing a new LTS branch yearly, and each LTS release will be supported for 3 years - 1 year of active maintenance with bugfixes, and 2 years of extended support for security fixes.
-
+A new LTS version will be created every 6 months.
+The lifetime of an LTS branch is currently planned for one year.
```

Comment:
We have 1 year of bug fix + 2 years of security fixes - we should highlight that.